meveo-model/src/main/java/org/meveo/model/security/Permission.java

```diff
@@ -84,6 +88,16 @@
     @JoinColumn(name = "category")
     private PermissionCategory category;
     
+    @Column(name = "code", nullable = false, length = 255)
```

Comment:
You don't need to add code & description as they are managed in BusinessEntity

Also, you can remove the "implementd IEntity<Long>"

Reply:
Rectified, deleted(code and description) from both files.